### ISSUE TITLE
GET / should deliver README

### ISSUE THREAD

**Issue by tillmo** [ramyadirsumilli/rest_owl_api#17]
A "GET /" should deliver a description of the API. An easy solution would be to deliver the contents of README.md, rendered in HTML. See http://stackoverflow.com/questions/7694887/is-there-a-command-line-utility-for-rendering-github-flavored-markdown how to do this.

**Comment by ramyadirsumilli** [comment]
Implemented. 
Please use ` ` `http://localhost:3000/ ` ` ` to get the HTML of README.md

**Comment by tillmo** [comment]
did you push the implementation? I cannot see anything.

**Comment by ramyadirsumilli** [comment]
Yes.. Could you try once again  with installation steps, i think only running `npm install` once again and then running the code will work.

**Comment by tillmo** [comment]
your last commit was 3 days ago. This issue has been filed yesterday. Unless you have a time machine, it won't be solved.

**Comment by ramyadirsumilli** [comment]
I am sorry.. My previous commits failed and I did not check it. 
Please re-verify now.

**Comment by tillmo** [comment]
good, now it works! Still one thing: could you turn the URLs into clickable links?

**Comment by ramyadirsumilli** [comment]
I worked over the clickable URLs, but not able to use POST actions.

**Comment by tillmo** [comment]
I now get
```
events.js:154
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE owlapi.hets.eu
```

**Comment by ramyadirsumilli** [comment]
Oh. I though you wanted me to edit the code to listen on owlapi.hets.eu ?

**Comment by tillmo** [comment]
no, the idea was to replace in README.md (but not elsewhere) localhost:3000 by owlapi.hets.eu.

**Comment by ramyadirsumilli** [comment]
Sorry. 
I performed the push again. 
Please check now.
